mappings/net/minecraft/entity/ai/AiGoalTrackTarget.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS xu net/minecraft/entity/ai/AiGoalTrackTarget
+	FIELD a checkCanNavigate Z
+	FIELD b canNavigateFlag I
+	FIELD c checkCanNavigateCooldown I
+	FIELD d timeWithoutVisibility I
+	FIELD e entity Lus;
+	FIELD f checkVisibility Z
+	FIELD g target Luk;
+	FIELD h maxTimeWithoutVisibility I
```

Comment:
I tend to make things more verbose when in doubt. I'd rather have a long name than a confusing one. I'm open to ideas for shorter names though.